old_scripts/odk_fieldmap_original/utils/dl_attachments.py

```diff
@@ -0,0 +1,62 @@
+# Copyright (c) 2020, 2021, 2022 Humanitarian OpenStreetMap Team
```

Comment:
As Rob pointed out, these years are wrong.

Reply:
While it's possible to fix all the copyright dates using a simple sed script, I usually just do it for the files I'm working on.